InternalDocs/frames.md

```diff
@@ -1,36 +1,33 @@
-# The Frame Stack
-
-Each call to a Python function has an activation record,
-commonly known as a "frame".
-Python semantics allows frames to outlive the activation,
-so they have (before 3.11) been allocated on the heap.
-This is expensive as it requires many allocations and
-results in poor locality of reference.
-
-In 3.11, rather than have these frames scattered about memory,
-as happens for heap-allocated objects, frames are allocated
-contiguously in a per-thread stack.
-This improves performance significantly for two reasons:
-* It reduces allocation overhead to a pointer comparison and increment.
-* Stack allocated data has the best possible locality and will always be in
-  CPU cache.
-
-Generator and coroutines still need heap allocated activation records, but
-can be linked into the per-thread stack so as to not impact performance too much.
+# Frames
 
-## Layout
-
-Each activation record consists of four conceptual sections:
+Each call to a Python function has an activation record, commonly known as a
+"frame". It contains information about the function being executed, consisting
+of four conceptual sections:
 
 * Local variables (including arguments, cells and free variables)
-* Evaluation stack
+* Evaluation stack and instruction pointer
```

Comment:
I don't think the instruction pointer belongs here (at least not conceptually).
I would add it to specials/linkage

Reply:
How about I change this line to
Execution state: evaluation stack, instruction pointer ?

It feels like instruction pointer is internal to the frame's working, rather than linking to external frames or objects.